src/helpers/index.ts

```diff
@@ -150,10 +121,12 @@ export function insertAfter<T>(collection: T[], afterItem: T, newItem: T) {
  * @param beforeItem
  * @param newItem
  */
-export function insertBefore<T>(collection: T[], beforeItem: T, newItem: T) {
-    const index = collection.indexOf(beforeItem);
+export function insertBefore<T>(collection: ArrayLike<T>, beforeItem: T, newItems: T | T[]) {
```

Comment:
We need use `ArrayLike` type now because `ts.NodeArray` now is `ReadonlyArray`.

![image](https://user-images.githubusercontent.com/6076919/35179712-770f12b4-fdda-11e7-9cb1-e59ad3f89459.png)